server.py

```diff
@@ -0,0 +1,108 @@
+from sys import argv
+
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from http.cookies import SimpleCookie
+from cgi import FieldStorage
+
+from jinja2 import Environment, FileSystemLoader
+
+
+HOST = '127.0.0.1'
+PORT = 8001
+
+if len(argv) > 1:
+    PORT = int(argv[1])
+
+
+class Handler(BaseHTTPRequestHandler):
+
+    def _get_cookies(self):
```

Comment:
minor: `_get/set_cookies()` would be more sensible if named `_get/set_auth_cookie()`

Reply:
fixed